.github/workflows/test.yml

```diff
@@ -398,17 +446,53 @@ jobs:
             --metas src/DDS-Router/.github/workflows/test_colcon.meta \
             --mixin coverage-gcc asan-gcc
 
-      - name: Run tests
+      - name: Run tests ddsrouter_core
+        run: |
+          source install/setup.bash && \
+          colcon test \
+            --packages-select ddsrouter_core \
+            --event-handlers=console_direct+ \
+            --return-code-on-test-failure \
+            --ctest-args \
+              --label-exclude xfail \
```

Comment:
Be careful, these tests are suppposed to execute flaky tests, so label `xfail` must be included (`regex`) and not excluded